dockstore-webservice/src/main/java/io/dockstore/webservice/languages/WDLHandler.java

```diff
@@ -295,39 +295,48 @@ public VersionTypeValidation validateTestParameterSet(Set<SourceFile> sourceFile
         }
 
         for (String importPath : currentFileImports) {
-            if (!imports.containsKey(importPath)) {
+            String absoluteImportPath = convertRelativePathToAbsolutePath(currentFilePath, importPath);
+            if (!imports.containsKey(absoluteImportPath)) {
                 SourceFile importFile = new SourceFile();
-                String absoluteImportPath = convertRelativePathToAbsolutePath(currentFilePath, importPath);
 
                 final String fileResponse = sourceCodeRepoInterface.readGitRepositoryFile(repositoryId, fileType, version, absoluteImportPath);
                 if (fileResponse == null) {
                     SourceCodeRepoInterface.LOG.error("Could not read: " + absoluteImportPath);
                     continue;
                 }
                 importFile.setContent(fileResponse);
-                importFile.setPath(importPath);
+                importFile.setPath(relativePath(currentFilePath, importPath));
                 importFile.setType(DescriptorLanguage.FileType.DOCKSTORE_WDL);
                 importFile.setAbsolutePath(absoluteImportPath);
-                imports.put(importFile.getPath(), importFile);
+                imports.put(absoluteImportPath, importFile);
                 imports.putAll(processImports(repositoryId, importFile.getContent(), version, sourceCodeRepoInterface, imports, absoluteImportPath));
             }
+
         }
         return imports;
     }
 
+    private String relativePath(String importingFile, String importedFile) {
```

Comment:
Not sure what this is going (Javadoc)

Alternatively, library method?
https://docs.oracle.com/javase/7/docs/api/java/nio/file/Path.html#relativize(java.nio.file.Path)
Or poke around Apache FilenameUtils

Reply:
This was mistakenly left over from my first attempts at fixing this; will remove entirely.